texastribune/donations
# Start tracking footer links and fix bg bug

#### What's this PR do?

Mirrors the donations footer to the main site's footer. Also fixes a tiny little missing pixel in the patterned background on the homepage.

#### Why are we doing this? How does it help us?

Keep our data consistent.

#### How should this be manually tested?

- Spin up locally and make sure every link in the footer of https://www.texastribune.org/ is also replicated on http://local.texastribune.org/donate
- Make sure you don't see a little gray line in the background image under the "show your support" button
- @emilyyount, you might want to peek at those `ga-` attributes as well

#### How should this change be communicated to end users?

N/A

#### Are there any smells or added technical debt to note?

N/A

#### What are the relevant tickets?

https://3.basecamp.com/3098728/buckets/736178/todos/1815979068

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***


#### TODOs / next steps:
